packages/tpetra/core/test/CrsMatrix/CrsMatrix_MatvecFence.cpp

```diff
@@ -151,7 +151,11 @@ namespace {
     using namespace Tpetra::Details;
     FenceCounter::reset();
     FenceCounter::start();
+#if KOKKOS_VERSION >= 40599
```

Comment:
``iter_num`` controls both the expected number of fences and how many times the loop below is run. We still want to run the loop and know that the number of fences doesn't increase unintentionally later, so the expected number of fences below is what this PR should change.

Reply:
yeah, I thought I force pushed the change to the correct spot. sorry, I'll push the update now

Reply:
@brian-kelley I force-pushed a fix in [a8f784d](https://github.com/trilinos/Trilinos/pull/13854/commits/a8f784db5339d9acdfb422a151fa88d622639a25) , sorry about that